include/lsst/afw/geom/Extent.h

```diff
@@ -353,6 +356,98 @@ typedef Extent<double,2> ExtentD;
 typedef Extent<double,2> Extent2D;
 typedef Extent<double,3> Extent3D;
 
+/**
+ *  Return the component-wise truncation (round towards zero).
+ *
+ *  In Python, this is available as both a free function and a method on ExtentD.
+ */
+template <int N>
+Extent<int,N> truncate(Extent<double,N> const & input);
+
+/**
+ *  Return the component-wise floor (round towards more negative).
+ *
+ *  In Python, this is available as both a free function and a method on ExtentD.
+ */
+template <int N>
+Extent<int,N> floor(Extent<double,N> const & input);
+
+/**
+ *  Return the component-wise floor (round towards more positive).
+ *
+ *  In Python, this is available as both a free function and a method on ExtentD.
+ */
+template <int N>
+Extent<int,N> ceil(Extent<double,N> const & input);
```

Comment:
This should probably have an accompanying test case.

I don't see any tests for `truncate()` and `floor()` either, but they are used in the definitions of operators which are tested, so we can be reasonably confident they work. I don't think `ceil()` is used anywhere.